The following patch reverts this change as it breaks gnome with x11 which
is still the default with void linux:

From 55093e304ec0c8cdc287ed933f4d78ca6eea5e07 Mon Sep 17 00:00:00 2001
From: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
Date: Mon, 29 Feb 2016 19:50:43 +0100
Subject: [PATCH] backend: check for wayland before x11

this way, clutter-gtk applications work fine on wayland
without the need of exporting CLUTTER_BACKEND=wayland
as gtk+ checks for wayland and x11 in the same order

https://bugzilla.gnome.org/show_bug.cgi?id=762888

Signed-off-by: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
---
 clutter/clutter-backend.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/clutter/clutter-backend.c b/clutter/clutter-backend.c
index d150e7c..b4e5f82 100644
--- clutter/clutter-backend.c
+++ clutter/clutter-backend.c
@@ -545,12 +545,12 @@ static const struct {
 #ifdef CLUTTER_WINDOWING_GDK
   { CLUTTER_WINDOWING_GDK, clutter_backend_gdk_new },
 #endif
+#ifdef CLUTTER_WINDOWING_X11
+  { CLUTTER_WINDOWING_X11, clutter_backend_x11_new },
+#endif
 #ifdef CLUTTER_WINDOWING_WAYLAND
   { CLUTTER_WINDOWING_WAYLAND, clutter_backend_wayland_new },
 #endif
-#ifdef CLUTTER_WINDOWING_X11
-  { CLUTTER_WINDOWING_X11, clutter_backend_x11_new },
-#endif
 #ifdef CLUTTER_WINDOWING_EGL
   { CLUTTER_WINDOWING_EGL, clutter_backend_egl_native_new },
 #endif
